doc/man1/req.pod

```diff
@@ -255,6 +256,15 @@ be a positive integer. The default is 30 days.
 Serial number to use when outputting a self signed certificate. This
 may be specified as a decimal value or a hex value if preceded by B<0x>.
 
+=item B<-extension ext>
+
+Add a specific extension to the certificate (if the B<-x509> option is
+present) or certificate request.  The argument must have the form of
+config key/value pairs (i.e. the lines you commonly find in the config
+file).
```

Comment:
I'd be more explicit with "key/value pairs *as* they would appear in a config file".

Reply:
Also you could replace `key/value pairs` by `key=value pairs`.

Reply:
Oh! It appears that it should be "pair", i.e. singular.

Reply:
And same holds for "lines".

Reply:
Thank you.  Will fix